numpy/doc/constants.py

```diff
@@ -296,8 +296,9 @@ def add_newdoc(module, name, doc):
     See Also
     --------
     isnan : Shows which elements are Not a Number.
+
     isfinite : Shows which elements are finite (not one of
-               Not a Number, positive infinity and negative infinity)
+    Not a Number, positive infinity and negative infinity)
```

Comment:
Out of curiousity, where does this show up in the documentation? I cannot find it, either interactively or in the html.

Reply:
I think the purpose of these docs is to support:
```
import numpy.doc

help(numpy.doc.constants)
```

which strikes me as a pretty weird usage